modules/parent-join/src/main/java/org/elasticsearch/join/mapper/ParentIdFieldMapper.java

```diff
@@ -123,11 +121,6 @@ public Object valueForDisplay(Object value) {
             BytesRef binaryValue = (BytesRef) value;
             return binaryValue.utf8ToString();
         }
-
-        @Override
-        public Query existsQuery(QueryShardContext context) {
-            return new DocValuesFieldExistsQuery(name());
-        }
     }
```

Comment:
It's annoying that neither this nor ParentJoinFieldMapper actually have tests for existsQuery, but they'll get them once we convert their test classes to MapperTestCase and it works by inspection now.

Reply:
agreed

Reply:
I opened https://github.com/elastic/elasticsearch/issues/62774 to convert all the remaining test cases